### PR TITLE
mavsdk 0.22.0 (new formula)

### PR DESCRIPTION
**Pull request by julianoes**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by julianoes** [comment]
Is anything needed here from my side to move this along? Thanks.

**Comment by chenrui333** [comment]
@julianoes, I think you are doing wonderful, it is just slow for us to do the merge during holidays.

**Comment by chenrui333** [comment]
@BrewTestBot test this please

File: Formula/mavsdk.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  desc "API and library for MAVLink compatible systems written in C++11"
+  homepage "https://mavsdk.mavlink.io"
+  url "https://github.com/mavlink/MAVSDK.git",
+    :tag      => "v0.22.0",
+    :revision => "164e5fe719608898b009364dbd870846d5163bfc"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", "-Bbuild/default",
```
Comment (fxcoudert) [inline Formula/mavsdk.rb]: This should use `std_cmake_args`

File: Formula/mavsdk.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
+class Mavsdk < Formula
+  desc "API and library for MAVLink compatible systems written in C++11"
+  homepage "https://mavsdk.mavlink.io"
+  url "https://github.com/mavlink/MAVSDK.git",
```
Comment (fxcoudert) [inline Formula/mavsdk.rb]: If possible we prefer to build from source tarballs, rather than git checkout.

File: Formula/mavsdk.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      int main() {
+          mavsdk::Mavsdk mavsdk;
+          mavsdk.version();
+          mavsdk::System& system = mavsdk.system();
+          auto info = std::make_shared<mavsdk::Info>(system);
+          return 0;
+      }
+    EOS
+    system ENV.cxx, "-std=c++11", testpath/"test.cpp", "-o", "test",
+                  "-I#{include}/mavsdk",
```
Comment (fxcoudert) [inline Formula/mavsdk.rb]: Wrong indentation for continuation lines

**Comment by fxcoudert** [comment]
@julianoes Can you please open a PR to adjust in response to the review comments, please?

@chenrui333 Let's try to keep the style of formulas homogeneous, it makes them easier to maintain in the long run

**Pull request closed without merging**